eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -45,26 +45,37 @@ jobs:
           OSName: 'Linux'
           OSVmImage: 'ubuntu-16.04'
           PythonVersion: '2.7'
+          CoverageArg: ''
```

Comment:
Can this be setup as a global variable and then only override it in the case you want to set it? That would help reduce the redundancy.